nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -169,6 +175,23 @@ public NullnessStore getLocalVarInfoBefore(TreePath path, Context context) {
                   || e.getKind().equals(ElementKind.LOCAL_VARIABLE);
             }
           }
+
+          // a filter for Optional get() call
```

Comment:
Why are we adding code to the core here? I thought the idea was to abstract this all behind a handler and guard it with a flag. This code is neither... am I missing something?

Reply:
Moved it to the handler 👍